charts/actions-runner-controller/crds/actions.summerwind.dev_runners.yaml

```diff
@@ -5245,6 +5231,31 @@ spec:
                     - expiresAt
                     - token
                   type: object
+                workflow:
+                  description: WorkflowStatus contains various information that is propagated from GitHub Actions workflow run environment variables to ease monitoring workflow run/job/steps that are triggerred on the runner.
```

Comment:
FYI, `make manifests` (or controller-tools `controller-gen` used internally by the make target) automatically reflects the Go field comment to this description field.